index.php

```diff
@@ -5,6 +5,4 @@
 $db = getDB();
 $data = getData($db);
```

Comment:
Is it possible, as you had suggested to me, that, by assigning the variables here, a user could reassign the variable; therefore, is it possible to pass these to index as purely functions or already set variables?

Reply:
I could maybe set them as constants but probably shouldn't bundle functions together.
Want to keep my functions file just for setting functions.

Do you have any suggestions?

Reply:
I have had a look and a suggestion is to put them in a class and declare them as private, and instead using getters for attaining the information.